js/language_server/src/index.js

```diff
@@ -0,0 +1,68 @@
+const { XMLHttpRequest } = require("../src/XMLHttpRequest.js");
+
+global.XMLHttpRequest = XMLHttpRequest;
+
+const SemgrepEngineWasm = require("../dist/semgrep-engine");
+
+import { getDirname } from "cross-dirname";
+
+const WASM_FILENAME = "semgrep-engine.wasm";
```

Comment:
we're using wasm now?

Reply:
We have been for turbo mode. PCRE + Libyaml are compiled to wasm